supporter-product-data/src/main/scala/com/gu/lambdas/ProcessSupporterRatePlanItemLambda.scala

```diff
@@ -91,14 +95,22 @@ object ProcessSupporterRatePlanItemLambda extends SafeLogging {
 class ContributionAmountFetcher(config: ZuoraQuerierConfig) extends SafeLogging {
   lazy val zuoraService = new ZuoraSubscriptionService(config, configurableFutureRunner(60.seconds))
 
-  def fetchContributionAmountFromZuora(supporterRatePlanItem: SupporterRatePlanItem) =
+  private def contributionAmountFromZuoraSubscription(subscription: MinimalZuoraSubscription) =
+    for {
+      contributionRatePlan <- subscription.ratePlans.find(ratePlan => contributionIds.contains(ratePlan.id))
+      charges <- contributionRatePlan.ratePlanCharges.headOption
```

Comment:
Just to confirm - it's always the first rate plan charge we care about here?

Reply:
Yes there's only ever one charge on a contribution rate plan